layouts/partials/footer.html

```diff
@@ -0,0 +1,52 @@
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="https://stackoverflow.com/tags/rapids" target="_blank" ><i class="fa-brands fa-stack-overflow"></i> Stack Overflow</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="https://twitter.com/rapidsai" target="_blank" ><i class="fa-brands fa-twitter"></i> Twitter</a>
+    </div>
+    <br>
+    <div class="footer-row has-text-centered">
+        <a class="footer-item" href="/"><i class="fa-regular fa-house-blank"></i> Home</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="/ecosystem"><i class="fa-light fa-chart-network"></i> Ecosystem</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="/learn-more"><i class="fa-regular fa-circle-info"></i> Learn More</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="/news"><i class="fa-light fa-newspaper"></i> News</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="https://docs.rapids.ai/user-guide" target="_blank" ><i class="fa-light fa-hand-point-right"></i> User Guides</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="https://docs.rapids.ai/install" target="_blank" ><i class="fa-light fa-download"></i> Install</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="https://docs.rapids.ai/" target="_blank" ><i class="fa-light fa-file-code"></i> Docs</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="https://github.com/rapidsai" target="_blank" ><i class="fa-brands fa-github"></i> GitHub</a>
+    </div>
+    <div class="footer-row has-text-centered">
+        <a class="footer-item" href="/featured"><i class="fa-light fa-gallery-thumbnails"></i> Featured</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="https://medium.com/rapids-ai" target="_blank"><i class="fa-regular fa-message-quote"></i> Blog Posts</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="https://docs.rapids.ai/notices" target="_blank"><i class="fa-light fa-triangle-exclamation"></i> Service Notices</a>
+    </div>
+    <div class="footer-row has-text-centered">
+        <a class="footer-item" href="/branding"><i class="fa-regular fa-object-group"></i> Branding and Guides</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="#"><i class="fa-light fa-file-invoice"></i> Terms</a>
+        <span class="footer-item"> | </span>
+        <a class="footer-item" href="#"><i class="fa-light fa-face-disguise"></i> Privacy</a>
+    </div>
+
+    <div class="footer-row has-text-centered has-text-white"><br>
+        <strong>© Copyright RAPIDS {{ now.Format "2006" }}</strong>
```

Comment:
We typically label our library documentation authorship and copyrights ownership as "NVIDIA Corporation."
```suggestion
        <strong>© Copyright NVIDIA Corporation {{ now.Format "2006" }}</strong>
```